### ISSUE TITLE
default logging.threshold not available in laxar-compatibility.with-deps.js

### ISSUE THREAD

**Issue by jpommerening** [LaxarJS/laxar#424]
```
  Error: Assertion error: Expected value to be defined and not null. Details: Configuration is missing mandatory entry: logging.threshold
```

Probably caused by accessing `ax.configuration.get` without `bootstrap()`-ing a LaxarJS application.
Noticed this when testing the laxar-input-control…

**Comment by jpommerening** [comment]
Hmm… seems like somethings wrong with webpack here. Investigating…

**Comment by jpommerening** [comment]
The problem seems to occur because _something_ is setting `window.laxar` to be the Laxar module, causing the configuration defaults ([here](https://github.com/LaxarJS/laxar/blob/936fb2c/laxar-compatibility.js#L88)) to be ignored.

**Comment by jpommerening** [comment]
I'm speculating where the global `window.laxar` could come from… maybe this could be the way webpack passes around "externalized" dependencies? `laxar-mocks` depends on `laxar` and is required before the spec itself requires `laxar`. I aliased it to resolve to `laxar/dist/laxar-compatibility.with-deps`, but at that point `window.laxar` is already set to be the `laxar` module…

If `laxar-compatibility` is the only place where we evaluate `window.laxar` I'd try to add a small piece of code to the main `laxar` entry point that exports the value of `window.laxar` if it is set. Then, in `laxar-compatibility` I'd check if `window.laxar` has the `__esModule` property and in that case access `laxar._hiddenGlobalConfig` or something like that…

Testing this approach I discovered another problem: I have two `laxar` modules in my `node_modules` directory:
- `node_modules/laxar`
- `node_modules/laxar-uikit/node_modules/laxar`
Naturally, webpack decided to resolve laxar-uikit's dependency on laxar with the module nested inside `laxar-uikit`. So after doing all of the above there was still _another_ call of `configuration.ensure('logging.threshold')` that caused the same error as before. Only after deleting the nested module, did all call sites use the same instance of the laxar-compatiblity module …

**Comment by jpommerening** [comment]
After a bit more trial-and-error I still can't reliably access `window.laxar`. My previous attempt evaluated to "undefined" every time which helped in this case, but is not what it's supposed to do.

**Comment by jpommerening** [comment]
Seems the `window.laxar` module was actually the `laxar/polyfills` module, loaded by karma as a dependency for the other modules. Filed issue #426 to resolve this…

**Comment by jpommerening** [comment]
fixed via #426
